### ISSUE TITLE
markdownlint-cli does not respect default node installation in system

### ISSUE THREAD

**Issue by Tom-Notch** [igorshubovych/markdownlint-cli#410]
Check [this similar issue](https://github.com/jupyterlab/jupyterlab/issues/12675)
This happens specifically on my arm64 edge device installed with ubuntu 18.04:
```Shell
An unexpected error has occurred: CalledProcessError: command: ('/external/home/airlab/.cache/pre-commit/repomv4gd62l/node_env-default/bin/node', '/external/home/airlab/.cache/pre-commit/repomv4gd62l/node_env-default/bin/npm', 'install', '--dev', '--prod', '--ignore-prepublish', '--no-progress', '--no-save')
return code: 1
stdout: (none)
stderr:
    /external/home/airlab/.cache/pre-commit/repomv4gd62l/node_env-default/bin/node: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.28` not found (required by /external/home/airlab/.cache/pre-commit/repomv4gd62l/node_env-default/bin/node)
Check the log at /external/home/airlab/.cache/pre-commit/pre-commit.log
```

**Comment by DavidAnson** [comment]
Everything in that error seems to point to pre-commit. This project does not try to install Node.js, it just runs under whatever is installed (assuming the installed Node runs successfully).

**Comment by DavidAnson** [comment]
The issue you linked says basically the same thing, but with more detail.
